packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -562,11 +566,7 @@ export class AtomicSearchBox implements InitializableComponent<Bindings> {
             : 'suggestions-single-list'
         }`}
         class={`flex w-full z-10 absolute left-0 top-full rounded-md bg-background border border-neutral ${
-          this.suggestionManager.hasSuggestions &&
-          this.isExpanded &&
-          !this.isSearchDisabledForEndUser(this.searchBoxState.value)
-            ? ''
-            : 'hidden'
+          this.shouldShowSuggestions ? '' : 'hidden'
```

Comment:
Should there be an `aria-hidden` added on the englobing div then ?

Reply:
Hm you bring a very good point, my fix is incomplete:
I need the suggestion that `div` to be in the DOM **and** visible but empty.
The rationale is that an Element is not in the accessibility tree in all four of this cases:
- The HTML [hidden](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) attribute is present
- The element or the element's ancestor is hidden with [display: none](https://developer.mozilla.org/en-US/docs/Web/CSS/display)
- The element or the element's ancestor is hidden with [visibility: hidden](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility)
- The HTML [aria-hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden) attribute is set to true.

Source: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden